makeicons/pages.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+This file is just the bok_choy pages file for our icons. bok_choy
+is a test framework, so we follow this split into pages/icon, although
+it is a little artifical in this case.
+"""
+from bok_choy.page_object import PageObject
+
+
+class IconsPage(PageObject):
```

Comment:
Why do you still need these icons? These aren't the ones I'm seeing when I test the PR, correct?